Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -868,9 +850,52 @@ public void MarkAsDirty()
             LastChanged = DateTime.UtcNow;
         }
 
-        public bool CheckSystemValidity()
+        public ItSystemUsageValidationResult CheckSystemValidity()
+        {
+            var errors = new List<ItSystemUsageValidationError>();
+
+            var today = DateTime.UtcNow.Date;
+
+            var dateErrors = CheckDatesValidity(today).ToList();
+            var validAccordingToStatus = CheckLifeCycleValidity();
+            //If either one of the conditions pass the system is considered active
+            var isValid = dateErrors.Any() == false && validAccordingToStatus.IsNone;
+
+            errors.AddRange(dateErrors);
+            if (validAccordingToStatus.HasValue)
+            {
+                errors.Add(validAccordingToStatus.Value);
+            }
+
+            return new ItSystemUsageValidationResult(errors, isValid);
```

Comment:
in stead of isValid just use `errors.Any() == false` .. or just derive the value in the ctor since the rule is now clear: `Any error means "invalid"`